### PR TITLE
Adding-Resources

### PR DESCRIPTION
**Pull request by ShivamPathak99**
This reverts commit 15926b9682ac0b713e06fb1c7b0128b87022fb1b.

## What does this PR do?
Add resource(s) |  | Improve repo

## For resources
I want to add the most recent edition of official Raspberry Pi Handbook and Beginner's Guide 4th Edition. 

### Description
Both of them consists of collection of Inspirational projects which totally belongs to DIY category and a useful guide for starting with RaspberryPi.

### Why is this valuable (or not)?
Yes, it will definitely add value to repo. These handbooks have been prepared by the makers of the MagPi that is the official Raspberry Pi magazine. Here the ideas has been conveyed in very creative way, the authors have used attractive photos and interactive design for quickly dragging the readers attention.

### How do we know it's really free?
It is clearly mentioned on the official TheMagPi Website that the soft copy of the Handbook is freely available to download.

### For book lists, is it a book? For course lists, is it a course? etc.
- [The Official Raspberry Pi Handbook 2023 (204-page PDF)](https://magpi.raspberrypi.com/books/handbook-2023)
- [Raspberry Pi Beginner's Guide 4th Edition (252-pages- PDF)](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up
- ***Please consider it for HactoberFest.***
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by ShivamPathak99 — COMMENTED** [review]
Please consider it for Hacktoberfest.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2078,8 +2078,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 
 ### Raspberry Pi
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:2080]:
```suggestion
### Raspberry Pi

```

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2078,8 +2078,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
-* [Raspberry Pi: Measure, Record, Explore](https://leanpub.com/RPiMRE/read) - Malcolm Maclean (HTML)
 * [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)
+* [Raspberry Pi: Measure, Record, Explore](https://leanpub.com/RPiMRE/read) - Malcolm Maclean (HTML)
 * [Raspberry Pi Users Guide (2012)](http://www.cs.unca.edu/~bruce/Fall14/360/RPiUsersGuide.pdf) - Eben Upton (PDF)
 * [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023)
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
```
[the remark this answers is not in the document]
Comment (ShivamPathak99) [reply books/free-programming-books-langs.md:2086]: I have been trying to resolve it  from yesterday but even after so many changes and several pull requests, it is kind of hard to understand what is happening. Can you tell me what is the real problem over here.

**Comment by ShivamPathak99** [comment]
Learned so many things about pull request and ways of handling errors.

**Comment by ShivamPathak99** [comment]
Please consider it for ***Hacktoberfest***.

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by eshellman** [comment]
these resource require? email addresses - are they required or optional?

**Comment by ShivamPathak99** [comment]
> > these resource require? email addresses - are they required or optional?
> 
> **It is completely optional, you can download it without giving any personal information. You can directly download the resource by clicking on the given link (_see the image file attached below_).** The Magpi

pls merge it

**Comment by ShivamPathak99** [comment]
Is there any other adjustments needed?

**Pull request closed without merging**